StRoot/StFttDbMaker/StFttDb.h

```diff
@@ -113,11 +115,29 @@ class StFttDb : public TDataSet {
 
     void getGloablOffset( UChar_t plane, UChar_t quad, float &dx, float &sx, float &dy, float &sy, float &dz, float &sz );
 
+    enum TimeCutMode {
+      CalibratedBunchCrossingMode = 0,
+      TimebinMode = 1
+    };
+
+    void setTimeCut(TimeCutMode mode, int low=-70, int high=30)
+    {
+        mTimeCutMode=mode; 
+        mTimeCutLow=low; 
+        mTimeCutHigh=high;
+    }
+
     void getTimeCut( StFttRawHit * hit, int &mode, int &l, int &h ){
-      mode = 0;
+      mode = mTimeCutMode;
+      l = mTimeCutLow;
+      h = mTimeCutHigh;
+      // mode = StFttDb::TimeCutMode::CalibratedBunchCrossingMode;
+      // l = -3;
+      // h = 3;
+      //mode = StFttDb::TimeCutMode::TimebinMode;
       // ideal; take +/- 3 bunch crossings
-      l = -3;
-      h = 3;
+      //l = -70;
+      //h = 30;
```

Comment:
This function `void getTimeCut( StFttRawHit * hit, int &mode, int &l, int &h )` takes a pointer to StFttRawHit. But I don't see it being used. If that is how the function is intended to work, shouldn't the argument be removed?

Reply:
You are right, somehow I lost the original DB based code that looks up a unique cut for each hit.
Fixed in 28da7017d2f9ed86b953205c21800711d2968e3e
In 34e97f28ccce8640e0823eadb939929b867a9b9a and 068ff9653e5458200175f7b790ecb85844eee436

I made sure that the default is production mode (read from DB) and that the flag mUserDefinedTimeCut is set whenever a time cut is applied by the user (via setTimeCut(...) )